db/write_thread.h

```diff
@@ -360,6 +360,14 @@ class WriteThread {
   // Enable pipelined write to WAL and memtable.
   const bool enable_pipelined_write_;
 
+  // The lower limit of maximum number of bytes that are written in a single
+  // batch of WAL or memtable write.
+  const uint64_t lower_limit_write_batch_group_size_bytes;
+
+  // The upper limit of maximum number of bytes that are written in a single
+  // batch of WAL or memtable write.
+  const uint64_t upper_limit_write_batch_group_size_bytes;
```

Comment:
Also think about a better name. Is `max_write_batch_group_size` better, for example?

Reply:
Updated the name to 'max_write_batch_group_size_bytes'.
Appended bytes in the name to specify unit of size.